edge/pkg/edged/edged.go

```diff
@@ -328,7 +336,16 @@ func (e *edged) Start() {
 		volumepathhandler.NewBlockVolumePathHandler(),
 	)
 	go e.volumeManager.Run(edgedutil.NewSourcesReady(e.isInitPodReady), utilwait.NeverStop)
-	go utilwait.Until(e.syncNodeStatus, e.nodeStatusUpdateFrequency, utilwait.NeverStop)
+
+	if features.DefaultMutableFeatureGate.Enabled(features.NodeLease) {
```

Comment:
Thanks for your feedback! Considering that some users may not want to be at the risk of using this new feature, because the cluster of using node status is running stably in their environment, we add a feature gate here instead of setting it as default feature.

Reply:
This feature has been GA for serveral Kubernetes release and is stable，sugget to keep with kubernetes feature PreRelease

Reply:
The implementation of kubernetes is stable, but we cannot use its code directly. The implementation in this pr is not stable enough.